src/Commands/MakeFactoryReloadedCommand.php

```diff
@@ -32,24 +40,45 @@ class MakeFactoryReloadedCommand extends GeneratorCommand
     /** @var string */
```

Comment:
The package already requires PHP 7.4 currently. So could already use property types here instead of the docblocks.

Reply:
I did that for the properties that we set, but left the inherited properties alone because PHPStorm didn't like me adding types. And only in the command. Haven't looked at any other files.